lib/signs/utilities/source_config.ex

```diff
@@ -1,29 +1,35 @@
-  show up on the lines of the sign. If two lists are provided, then the next trip from each list will
-  appear on different lines of the sign.
+    "id": "roxbury_crossing_southbound",
+    "headway_group": "orange_trunk",
+    "type": "realtime",
+    "pa_ess_loc": "OROX",
+    "text_zone": "s",
+    "audio_zones": ["s"],
+    "read_loop_offset": 60,
+    "source_config": [...]
+  },
+
+  * id: the internal ID of this "sign". Really, it's a "zone" and could have a few physical signs as a part of it in the station, but they all display exactly the same content. Logically they're a single entity in the ARINC system as far as we can interact with it. This ID is also used by signs-ui, and is how realtime-signs knows how signs-ui has configured the sign.
+  * headway_group: this references the signs-ui values that the PIOs set for headways.
+  * type: formerly there were different kinds of signs in realtime_signs, but now they're all "realtime" only.
+  * pa_ess_loc: the ARINC station code. Starts with the line color B, R, O, G, or M, and then three letters for the station.
+  * text_zone: one of the 6 zones ARINC divides a station into, for the purpose of sending text to the sign.
+  * audio_zones: a list of those ARINC zones for the purpose of sending audio. This may differ from the text_zone when the speakers are close enough to cause confusion. For example, at Park Street for the Red line, we have the center platform text_zone set to "c", but the audio_zones set to [], while the north zone we have the text_zone set to "n" and the audio_zones set to ["n", "c"]. So the center platform doesn't play audio of its *own*, but rather the north and south platforms play their audio over the center platform speakers. Since a train that goes to "ARR" will do so simultaneously for either the north or south platform *and* the center platform, this configuration prevents simultaneous, slightly overlapping audio of the same content.
+  * read_loop_offset: how many seconds to wait after app start-up before entering the "read loop", which reads things like "The next train to X arrives in 3 minutes. The following train arrives in 8 minutes". We generally have different read_loop_offsets for different zones, or where speakers are particularly close, to encourage them to play their audio at different times.
+  * source config: see below.
+
+  ## Source config
+  A sign's data is provided via the "source_config" key. This data is a list of lists of data sources, called "sources".
+
+  A list of sources can be provided: [{...}, {...}, {...}, ...]. All the sources will be sorted together in order of their respective arrivals or departures (depending on their respective "terminal" values). For example, JFK mezzanine northbound uses 70086 and 70096 in a source group, since those are the Ashmont and Braintree platforms, and it wishes to display the next train from either location.
```

Comment:
I'm not sure what "sorted together" means here, or the implications of sources being sorted vs. not; maybe we could explain it?